tests/src/integration/common/wskdeploy.go

```diff
@@ -18,38 +18,46 @@
 package common
 
 import (
-    "os"
-    "os/exec"
+	"os"
+	"os/exec"
 )
 
 const cmd = "wskdeploy"
 
 type Wskdeploy struct {
-    Path string
-    Dir  string
+	Path string
+	Dir  string
 }
 
 func NewWskdeploy() *Wskdeploy {
-    return NewWskWithPath(os.Getenv("GOPATH") + "/src/github.com/apache/incubator-openwhisk-wskdeploy/")
+	return NewWskWithPath(os.Getenv("GOPATH") + "/src/github.com/apache/incubator-openwhisk-wskdeploy/")
 }
 
 func NewWskWithPath(path string) *Wskdeploy {
-    var dep Wskdeploy
-    dep.Path = cmd
-    dep.Dir = path
-    return &dep
+	var dep Wskdeploy
+	dep.Path = cmd
+	dep.Dir = path
+	return &dep
 }
 
 func (wskdeploy *Wskdeploy) RunCommand(s ...string) ([]byte, error) {
-    command := exec.Command(wskdeploy.Path, s...)
-    command.Dir = wskdeploy.Dir
-    return command.CombinedOutput()
+	command := exec.Command(wskdeploy.Path, s...)
+	command.Dir = wskdeploy.Dir
+	return command.CombinedOutput()
 }
 
 func (wskdeploy *Wskdeploy) Deploy(manifestPath string, deploymentPath string) ([]byte, error) {
-    return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
 }
 
 func (wskdeploy *Wskdeploy) Undeploy(manifestPath string, deploymentPath string) ([]byte, error) {
-    return wskdeploy.RunCommand("undeploy", "-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("undeploy", "-m", manifestPath, "-d", deploymentPath)
+}
+
+func (wskdeploy *Wskdeploy) DeployProjectPathOnly(projectPath string) ([]byte, error) {
+        return wskdeploy.RunCommand("-p", projectPath)
+}
+
+func (wskdeploy *Wskdeploy) DeployManifestPathOnly(manifestpath string) ([]byte, error) {
+	return wskdeploy.RunCommand("-p", manifestpath)
```

Comment:
@pritidesai , thanks for your review, I'll change the test code accordingly. 
@mrutkows , we could set the interactive mute, so the deploy will go by default. I'll change the test code.